tmc-langs-util/src/main/java/fi/helsinki/cs/tmc/langs/util/ProjectType.java

```diff
@@ -0,0 +1,21 @@
+package fi.helsinki.cs.tmc.langs.util;
+
+import fi.helsinki.cs.tmc.langs.LanguagePlugin;
+import fi.helsinki.cs.tmc.langs.ant.AntPlugin;
+
+/**
+ * All the possible project types
+ */
+public enum ProjectType {
```

Comment:
I'll take a look at Guice and try to implement it during this sprint if we have time. 

Thanks for the advice!